odl/contrib/datasets/ct/examples/mayo_reconstruct.py

```diff
@@ -0,0 +1,48 @@
+"""Reconstruct Mayo dataset using FBP and compare to reference recon.
+
+Note that this example requires that Mayo has been previously downloaded and is
+stored in "mayo_folder".
```

Comment:
I prefer the more correct term "directory" which can be abbreviated to the even shorter "dir". "folder" is so windows.

Reply:
Also, this may be misunderstood as literally a directory called "mayo_folder". Thus, mention that it's the variable with that name which counts.